tests/flow/test_comprehension_functions.py

```
@@ -0,0 +1,454 @@
+    def test10_any_all_comprehension_acceptance(self):
+        # Reject ANY and ALL comprehensions that don't include a WHERE predicate.
+        try:
+            self.graph.query("RETURN any(x IN [1,2])")
+            assert(False)
+        except redis.exceptions.ResponseError as e:
+            # Expecting a type error.
+            self.env.assertIn("requires a WHERE predicate", str(e))
+
+        try:
+            self.graph.query("RETURN all(x IN [1,2])")
+            assert(False)
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)